tests/model-builder.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const assert = require('assert');
+const BaseModel = require('../lib/base-model');
+
+describe('test model builder', () => {
```

Comment:
```suggestion
describe('Model builder', () => {
```

Estás en la carpeta `tests`, todos sabemos que es un test......  Lo mismo más abajo que se vuelve a repetir..